src/types.rs

```diff
@@ -1057,6 +1057,16 @@ impl FromRedisValue for bool {
                     invalid_type_error!(v, "Response status not valid boolean");
                 }
             }
+            Value::Data(ref bytes) => {
+                let res = from_utf8(bytes)?.to_string();
+                if res == "1" {
+                    Ok(true)
+                } else if res == "0" {
```

Comment:
`Vec<u8>` can compare to byte literal `b"1"`, whose type is `&'static [u8; 1]`. No additional allocation needed 😊
```suggestion
                if bytes == b"1" {
                    Ok(true)
                } else if bytes == "0" {
```